Add Time to celebrate blog post (#1169)
updatecli/website: content/en/blog/2023/10/time_to_celebrate.adoc

---

> title: "Time to celebrate"

Reviewer: ```suggestion
title: "Time to celebrate!"
```

> date: 2023-09-02T06:00:00+01:00

Author: ```suggestion
date: 2023-10-02T12:00:00+01:00
```

draft: false
weight: 50
images: [""]
contributors: ["olblak"]
---

> = Time to celebrate

Reviewer: ```suggestion
= Time to celebrate!
```

Author: I wonder why I can commit but I can't add suggestion to batch as you teach me

Reviewer: @olblak you can add suggestions to batch only on the "files" tab, not on the "conversation" one.

Author: haaaa that explain, why I managed to  batch some suggestions :) 
Thank you

I really love seeing those two company names sponsoring me as they are evolving in totally different domains, and not even from the continent where I live.

Ok, I see you coming, they are both software companies, but nowadays what company is not a software company that needs to maintain Git repositories?

=== Prefect

Prefect is the company behind the successful OSS project of the same name **link:https://github.com/PrefectHQ/prefect[Prefect]**. Prefect is a workflow orchestration tool that empowers developers to build, observe, and react to data pipelines.

image::/images/sponsors/prefect.png["Prefect",align="center"]

=== Foresight Mining

Foresight Mining is currently busy building the next generation of IOT collaborative CAD systems.

image::/images/sponsors/foresightmining.jpeg["Foresight Mining",align="center"]

== So what's coming next?

As always it's hard to tell, but I guess finding the right balance between implementing crazy ideas, having fun, and fixing bugs we encounter as Updatecli usage grows.

> With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by Matthew Warman where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.

Reviewer: ```suggestion
With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by https://warman.io/[Matthew Warman] where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```

> With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by https://warman.io/[Matthew Warman] where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.

Author: ```suggestion
With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```

Adding link and make it bold to better see it in the page

> With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.

Author: ```suggestion
With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach combining plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (or any others existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```

> In version 0.62.0, Matthew also contributed the Terraform Autodiscovery plugin that allows to handle automatically Terraform project updates.

Reviewer: ```suggestion
In version 0.62.0, Matthew also contributed the Terraform Autodiscovery plugin allowing to automatically handle Terraform project updates.
```

Reviewer: I would also add a link on the terraform autodiscovery plugin (like the ones above for the miscellaneous TF plugins)

Author: ```suggestion
In version 0.62.0, Matthew also contributed the **link:https://www.updatecli.io/docs/plugins/autodiscovery/terraform/[Terraform Autodiscovery plugin]** that allows to handle automatically Terraform project updates.
```

> In version 0.62.0, Matthew also contributed the Terraform Autodiscovery plugin allowing to automatically handle Terraform project updates.

Author: ```suggestion
In version 0.62.0, Matthew also contributed the **link:https://www.updatecli.io/docs/plugins/autodiscovery/terraform/[Terraform Autodiscovery plugin]** allowing to automatically handle Terraform project updates.
```

> For example, we can run commands such as `updatecli diff --experimental` to see what would be changed in Terraform project containing a Terraform lock file.

Reviewer: ```suggestion
For example, we can run commands such as `updatecli diff --experimental` to see what would change in Terraform project containing a Terraform lock file.
```

> On my side, I am exploring the possibility of using any OCI registries such as DockerHub or ghcr.io to share and reuse Updatecli policies.

Reviewer: ```suggestion
On my side, I am exploring the possibility of using OCI registries such as DockerHub or ghcr.io to share and reuse Updatecli policies.
```

As the project evolves, we face more and more the need to run Updatecli at scale. And being able to easily share and reuse existing Update policies would be another milestone.

== Let's Connect

As usual, if you have any questions feel free to reach out via our **link:https://matrix.to/#/#Updatecli_community:gitter.im[chat]**

And do not forget to star our project on GitHub repository **link:https://github.com/updatecli/updatecli/stargazers[updatecli/updatecli]** as it's probably one of the easiest ways to show your appreciation.

== The People ❤️

> The Updatecli wouldn't stand here today with all the people who contributed to it.

Author: ```suggestion
The Updatecli wouldn't stand here today without all the people who contributed to it.
```
